mobilecoind-json/src/bin/main.rs

```diff
@@ -345,6 +345,17 @@ fn build_and_submit(
     req.set_sender_subaddress(subaddress_index);
     req.set_outlay_list(RepeatedField::from_vec(vec![outlay]));
     req.set_max_input_utxo_value(max_input_utxo_value);
+    match &transfer.change_subaddress {
```

Comment:
this should be `if let Some(subaddress) = transfer.change_subaddress {` (maybe with `.as_ref()`)